src/Symfony/Component/Intl/composer.json

```diff
@@ -24,11 +24,12 @@
         }
     ],
     "require": {
-        "php": ">=8.1"
+        "php": ">=8.1",
+        "symfony/deprecation-contracts": "^2.5|^3",
+        "symfony/emoji-transliterator": "^6.3"
```

Comment:
can this be optional?
i.e. when i need the `intl` component without the emoji support

Reply:
Not in 6.x, see the pull request description.